app/controllers/room_guests_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RoomGuestsController < ApplicationController
+  def create
+    RoomGuest.create!(room_guest_params)
```

Comment:
Great job with this controller and using strong params! Exactly what I would do. 👍